images/win/scripts/ImageHelpers/ChocoHelpers.ps1

```diff
@@ -29,4 +29,21 @@ function Choco-Install {
             }
         }
     }
+}
+
+function Get-ChocoPackageLatestVersion {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $PackageName,
+        [Parameter(Mandatory)]
+        [string] $TargetVersion
+    )
+
+    $filterQuery = "`$filter=(Id eq `'$PackageName`') and (IsPrerelease eq false) and (Version ge `'$TargetVersion`') and (Version lt `'$($TargetVersion + 1)`')"

```

Comment:
It's not necessary to escape single quotes.

Reply:
Thanks. Will update.

Reply:
Updated